client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -1,11 +1,88 @@
 .header--input {
   color: #ffffff;
   text-align: center;
+  margin-bottom: 2rem;
+}
+
+.clickable-tabs {
+  display: table;
+  border-spacing: 1rem;
+  width: 70rem;
+  position: relative;
+  bottom: 3rem;
+}
+
+.clickable-tab {
+  color: #ffffff;
+  border-style: solid;
+  border-color: #f0f0f0;
+  border-width: medium;
+  border-radius: 1rem;
+  background-color: inherit;
+  display: table-cell;
+  padding: 1.5rem 1rem;
+  font-size: 0.9em;
+  margin-top: -1rem;
+  margin-left: 1rem;
+  position: relative;
+  left: -3rem;
+  max-width: 11.2rem;
+  text-align: center;
+  vertical-align: middle;
+  min-height: 15rem;
+  line-height: 1.4rem;
+  cursor: auto;
+}
+
+.clickable-tab:hover {
+  transition: 0.4s;
```

Comment:
what is being transitioned? usually you define it like `transition: opacity 0.4s`